core/src/main/java/co/aikar/commands/Annotations.java

```diff
@@ -49,8 +49,16 @@
         this.manager = manager;
     }
 
-    String getAnnotationValue(AnnotatedElement element, Class<? extends Annotation> annoClass, int options) {
-        Annotation annotation = element.getAnnotation(annoClass);
+    private static boolean hasOption(int bitmaskOptions, int option) {
+        return (bitmaskOptions & option) == option;
+    }
+
+    @Override
+    String getAnnotationValue(AnnotatedElement object, Annotation annotation, int options) {
+        if (annotation == null) { // allow for using a simple #getAnnotation instead of handling more in the usage
```

Comment:
revert this please and the matching change. 

using .getAnnotation() will not be allowed anymore in ACF, everything must route through this code